docs/tutorials/01_KalmanFilterTutorial.py

```diff
@@ -166,14 +166,11 @@
 
 # %%
 # Thus the ground truth is generated and we can plot the result
-fig = plt.figure(figsize=(10, 6))
-ax = fig.add_subplot(1, 1, 1)
-ax.set_xlabel("$x$")
-ax.set_ylabel("$y$")
-ax.axis('equal')
-ax.plot([state.state_vector[0] for state in truth],
-        [state.state_vector[2] for state in truth],
-        linestyle="--")
+
```

Comment:
As this is the first place the plotter will likely be encountered, it's worth doing an intro. "Stone Soup has an in-built plotting class called `Plott...`  which renders ground truth, measurements and state estimates in a consistent format." Or something like that. "It can be accessed by ..."